src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -479,7 +479,12 @@ protected function initializeBundles()
      */
     protected function getContainerClass()
     {
-        return $this->name.ucfirst($this->environment).($this->debug ? 'Debug' : '').'ProjectContainer';
+        $name = $this->name.ucfirst($this->environment).($this->debug ? 'Debug' : '').'ProjectContainer';
+        if (!preg_match('/^[a-zA-Z_\x7f-\xff]/', $name)) { // class name must start with a letter or underscore
```

Comment:
wouldn't it be somehow considered as BC break? the default name (e.g. appDevDebugContainer) is used by e.g. symfony plugin for phpstorm?